pkg/scheduler/apis/config/v1beta1/conversion.go

```diff
@@ -46,6 +46,149 @@ func getPluginArgConversionScheme() *runtime.Scheme {
+		return err
+	}
+	if err := convertToInternalPluginSet(in.Permit, &out.Permit, s); err != nil {
+		return err
+	}
+	if err := convertToInternalPluginSet(in.PreBind, &out.PreBind, s); err != nil {
+		return err
+	}
+	if err := convertToInternalPluginSet(in.Bind, &out.Bind, s); err != nil {
+		return err
+	}
+	if err := convertToInternalPluginSet(in.PostBind, &out.PostBind, s); err != nil {
+		return err
+	}
+	return nil
+}
+
+func convertToInternalPluginSet(in *v1beta1.PluginSet, out *config.PluginSet, s conversion.Scope) error {
+	if in == nil {
+		return nil
+	}
+	if in.Enabled != nil {
+		in, out := &in.Enabled, &out.Enabled
+		*out = make([]config.Plugin, len(*in))
+		for i := range *in {
+			if err := Convert_v1beta1_Plugin_To_config_Plugin(&(*in)[i], &(*out)[i], s); err != nil {
+				return err
+			}
+		}
+	} else {
+		out.Enabled = nil
+	}
+	if in.Disabled != nil {
+		in, out := &in.Disabled, &out.Disabled
+		*out = make([]config.Plugin, len(*in))
+		for i := range *in {
+			if err := Convert_v1beta1_Plugin_To_config_Plugin(&(*in)[i], &(*out)[i], s); err != nil {
+				return err
+			}
+		}
+	} else {
+		out.Disabled = nil
+	}
+	return nil
+}
+
+func Convert_config_Plugins_To_v1beta1_Plugins(in *config.Plugins, out *v1beta1.Plugins, s conversion.Scope) error {
+	out.QueueSort = new(v1beta1.PluginSet)
+	if err := convertToExternalPluginSet(in.QueueSort, out.QueueSort, s); err != nil {
+		return err
+	}
+	out.PreFilter = new(v1beta1.PluginSet)
+	if err := convertToExternalPluginSet(in.PreFilter, out.PreFilter, s); err != nil {
+		return err
+	}
+	out.Filter = new(v1beta1.PluginSet)
+	if err := convertToExternalPluginSet(in.Filter, out.Filter, s); err != nil {
+		return err
+	}
+	out.PostFilter = new(v1beta1.PluginSet)
+	if err := convertToExternalPluginSet(in.PostFilter, out.PostFilter, s); err != nil {
+		return err
+	}
+	out.PreScore = new(v1beta1.PluginSet)
+	if err := convertToExternalPluginSet(in.PreScore, out.PreScore, s); err != nil {
+		return err
+	}
+	out.Score = new(v1beta1.PluginSet)
+	if err := convertToExternalPluginSet(in.Score, out.Score, s); err != nil {
+		return err
+	}
+	out.Reserve = new(v1beta1.PluginSet)
+	if err := convertToExternalPluginSet(in.Reserve, out.Reserve, s); err != nil {
+		return err
+	}
+	out.Permit = new(v1beta1.PluginSet)
+	if err := convertToExternalPluginSet(in.Permit, out.Permit, s); err != nil {
+		return err
+	}
+	out.PreBind = new(v1beta1.PluginSet)
+	if err := convertToExternalPluginSet(in.PreBind, out.PreBind, s); err != nil {
+		return err
+	}
+	out.Bind = new(v1beta1.PluginSet)
+	if err := convertToExternalPluginSet(in.Bind, out.Bind, s); err != nil {
+		return err
+	}
+	out.PostBind = new(v1beta1.PluginSet)
+	if err := convertToExternalPluginSet(in.PostBind, out.PostBind, s); err != nil {
+		return err
+	}
+	return nil
+}
+
+func convertToExternalPluginSet(in config.PluginSet, out *v1beta1.PluginSet, s conversion.Scope) error {
```

Comment:
Can you use `Convert_config_PluginSet_To_v1beta1_PluginSet` instead?

Reply:
done.